src/machine/board_pca10059.go

```diff
@@ -23,9 +23,9 @@ const (
 // ADC pins
 const (
 	ADC1 Pin = 2
-	ADC2 Pin = 4
-	ADC3 Pin = 29
-	ADC4 Pin = 31
+	ADC2 Pin = 29
+	ADC3 Pin = 31
+	ADC4 Pin = 4 // only available as a pad on downside
```

Comment:
Does this change have anything to do with nrfutil zip?
If not, please give us another PR so that we can understand the commit history better.

Also, do you have any reason for this change?
I did not understand the meaning of the following.

> move Pin 0.4 to the end of list since almost unreachable

Reply:
THis slipped in accidentely. Motivation was support for the `examples/adc` code. The `PIN 0.4` aka `4` of the pca10059 board is located on the button of the USB stick in a quite unaccessible manner.
See 
![Bildschirmfoto 2021-04-05 um 18 59 46](https://user-images.githubusercontent.com/1179001/113601157-45061800-9641-11eb-899c-6ec924cab893.png)

Reply:
I see.
Could you please separate the PRs for easier review?
Thank you.